DEVELOPER_GUIDE.md

```diff
@@ -227,15 +233,15 @@ This method can also be used to develop against the [full distribution of OpenSe
 
 ### Configure OpenSearch Dashboards for security
 
-*This step is only mandatory if you have the [`security` plugin](https://github.com/opensearch-project/security) installed on your OpenSearch cluster with https/authentication enabled.*
+_This step is only mandatory if you have the [`security` plugin](https://github.com/opensearch-project/security) installed on your OpenSearch cluster with https/authentication enabled._
 
 Once the bootstrap of OpenSearch Dashboards is finished, you need to apply some
 changes to the default [`opensearch_dashboards.yml`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/config/opensearch_dashboards.yml#L25-L72) in order to connect to OpenSearch.
 
 ```yml
-opensearch.hosts: ["https://localhost:9200"]
-opensearch.username: "admin" # Default username on the docker image
-opensearch.password: "admin" # Default password on the docker image
+opensearch.hosts: ['https://localhost:9200']
+opensearch.username: 'admin' # Default username on the docker image
+opensearch.password: 'admin' # Default password on the docker image
 opensearch.ssl.verificationMode: none
```

Comment:
I'd skip this change - I like most of the other markdown linting changes, but these examples in the yml file use double quotes.

Reply:
Changed back.